.github/actions/helm-dry-run/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Helm Dry Run
+description: >
+  Execute helm dry run on given chart
+
+inputs:
+  chart-dir:
+    description: 'Path to the folder holding Chart.yaml'
+    required: true
+  test-rancher-url:
+    description: 'Rancher URL for tests, tests are skipped if not filled'
+    required: true
+  test-rancher-access-key:
+    description: 'Rancher access key for tests, tests are skipped if not filled'
+    required: true
+  test-rancher-secret-key:
+    description: 'Rancher secret key for tests, tests are skipped if not filled'
+    required: true
+  test-cluster-name:
+    description: 'Name of the Rancher cluster name for tests'
+    required: true
+  test-namespace:
+    description: >
+      Name of the namespace in rancher for tests (should be unique in PRs),
+      tests are skipped if not filled.
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Setup rancher
+      uses: Alfresco/alfresco-build-tools/.github/actions/setup-rancher-cli@v1.9.2
+      with:
+        url: ${{ inputs.test-rancher-url }}
+        access-key: ${{ inputs.test-rancher-access-key }}
+        secret-key: ${{ inputs.test-rancher-secret-key }}
+        context: ${{ inputs.test-cluster-name }}
+    - name: Execute helm dry-run
+      if: steps.check.test == 'true'
+      env:
+        CHART_DIR: ${{ inputs.chart-dir }}
+        NAMESPACE: ${{ inputs.test-namespace }}
+      shell: bash
+      run: |
+        NAMESPACE_LOWERCASE=$(echo ${NAMESPACE} | tr "[:upper:]" "[:lower:]")
+        helm upgrade $NAMESPACE_LOWERCASE $CHART_DIR \
+          --install \
+          --set global.gateway.domain=example \
+          --set global.keycloak.clientSecret=$(uuidgen) \
```

Comment:
looks like those are specific to the chart you are testing, you probably want to have a new input to pass any additional arguments to helm cmd